modules/monitoring-querying-metrics-by-using-the-federation-endpoint-for-prometheus.adoc

```diff
@@ -0,0 +1,63 @@
+// Module included in the following assemblies:
+//
+// * monitoring/accessing-third-party-monitoring-apis.adoc
+
+:_content-type: PROCEDURE
+[id="monitoring-querying-metrics-by-using-the-federation-endpoint-for-prometheus_{context}"]
+= Querying metrics by using the federation endpoint for Prometheus
+
+You can use federation to query a cluster's scraped metrics from a network location outside the cluster. 
+To do so, access the Prometheus `/federate` endpoint for the cluster via an {product-title} route. 
+
+You can access the `/federate` endpoints for default platform monitoring and for user-defined projects.
+
+[WARNING]
+====
+A delay in retrieving metrics data occurs when you use federation. 
+This delay can affect the accuracy and timeliness of the scraped metrics.
+
+In addition, federation can degrade the performance and scalability of your cluster, especially if you use federation to try and retrieve large amounts of metrics data.
+
+For these reasons, use federation only when you want to retrieve a limited, aggregated data set.
```

Comment:
Less than 1,000 samples per request should be fine. We can also say that clients shouldn't query the federation too frequently (a ballpark number would be not more often than every 30s in general).

Reply:
Reworded and added your recommendations